### PR TITLE
Unique number 3 C# code

### PR DESCRIPTION
**Pull request by rutujadhanawade**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

I have added code for unique number.  As mentioned in the description of #988 , I have used  bitwise AND to find the unique element in O(n) time and constant extra space.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

ISSUE - #988

### CODE REVIEW

**Comment by rutujadhanawade** [comment]
@iamrajiv @atarax665 @Kajol-Kumari @plazzy99 please review this.

File: C-Sharp/other/Unique_Number_3.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
+//Unique_Number_III
+//Given an array that contains all elements occurring 3 times, but one occurs only once. Find that unique element.
+
+using System.IO;
+using System;
+
+class Neoalgo {
```
Comment (iamrajiv) [inline C-Sharp/other/Unique_Number_3.cs]: @rutujadhanawade Rename the class name as `Unique_Number_III`.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@rutujadhanawade Your program name should be `Unique_Number_III.cs` not `Unique_Number_3.cs`.

File: C-Sharp/other/Unique_Number_3.cs (hunk 1 above)
Comment (rutujadhanawade) [reply C-Sharp/other/Unique_Number_3.cs]: Done

**Review by iamrajiv — APPROVED** [review]

**Review by atarax665 — APPROVED** [review]

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Comment by rutujadhanawade** [comment]
@iamrajiv please merge it.

**Pull request merged**